packages/e2e/features/ui/components/storage/storage-browser/render-storage-browser.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Storage Browser with default environment
+
+  Background:
+    Given I'm running the example "ui/components/storage/storage-browser/default-auth"
+  
+  @react
+  Scenario: Renders storage browser after login
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I click the "Sign in" button
+    Then I see the "Sign Out" button
+    Then I see "Home"
+    Then I see the "s3://fileuploaderbucket121216-dev/public/*" button
+
+  @react
+  Scenario: Download file "test.txt"
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I click the "Sign in" button
+    Then I see the "Sign Out" button
+    Then I see "Home"
+    Then I see the "s3://fileuploaderbucket121216-dev/public/*" button
+    Then I click the "s3://fileuploaderbucket121216-dev/public/*" button
+    Then I see "test.txt"
+    Then I click the "Download public/test.txt" button
+    Then I verify the downloaded file "test.txt" has the contents "file contents"
```

Comment:
For some reason, this works when I do `yarn e2e dev` and then run it manually through the UI, but doesn't work when I'm just using the CLI. When running it through CLI,  I'm able to see the file get downloaded but it's not able to read it I'm guessing?